driver/src/main.rs

```diff
@@ -90,9 +91,8 @@ struct Options {
 fn main() {
     let options = Options::from_args();
 
-    let optimization_model_string: String =
-        env::var("SOLVER_TYPE").unwrap_or_else(|_| String::from("NaiveSolver"));
-    let optimization_model = SolverType::from(optimization_model_string.as_str());
+    let (_, _guard) = logging::init(&options.log_filter);
+    info!("using options: {:#?}", options);
```

Comment:
I wonder if options it’s too vague here for a long statement. It looks like these correspond to runtime arguments, could we say something like that.

Reply:
I’ll open a PR for that, it’s already in master and would be an unrelated change.